src/main/java/gregtech/loaders/MaterialInfoLoader.java

```diff
@@ -14,415 +15,442 @@
+                new MaterialStack(Rubber, OrePrefix.plate.getMaterialAmount(Rubber) * 2), // plate (for the cable)
+                new MaterialStack(Wood, OrePrefix.plate.getMaterialAmount(Wood) * 2))); // plate
+
+        OreDictUnifier.registerOre(MetaTileEntities.HULL[GTValues.LV].getStackForm(), new ItemMaterialInfo(
+                new MaterialStack(Steel, OrePrefix.plate.getMaterialAmount(Steel) * 8), // plate
+                new MaterialStack(Tin, OrePrefix.cableGtSingle.getMaterialAmount(Tin) * 2), // single cable
+                new MaterialStack(Rubber, OrePrefix.plate.getMaterialAmount(Rubber) * 2), // plate (for the cable)
+                new MaterialStack(WroughtIron, OrePrefix.plate.getMaterialAmount(WroughtIron) * 2))); // plate
+
+        OreDictUnifier.registerOre(MetaTileEntities.HULL[GTValues.MV].getStackForm(), new ItemMaterialInfo(
+                new MaterialStack(Aluminium, OrePrefix.plate.getMaterialAmount(Aluminium) * 8), // plate
+                new MaterialStack(Copper, OrePrefix.cableGtSingle.getMaterialAmount(Copper) * 2), // single cable
+                new MaterialStack(Rubber, OrePrefix.plate.getMaterialAmount(Rubber) * 2), // plate (for the cable)
+                new MaterialStack(WroughtIron, OrePrefix.plate.getMaterialAmount(WroughtIron) * 2))); // plate
+
+        OreDictUnifier.registerOre(MetaTileEntities.HULL[GTValues.HV].getStackForm(), new ItemMaterialInfo(
+                new MaterialStack(StainlessSteel, OrePrefix.plate.getMaterialAmount(StainlessSteel) * 8), // plate
+                new MaterialStack(Gold, OrePrefix.cableGtSingle.getMaterialAmount(Gold) * 2), // single cable
+                new MaterialStack(Rubber, OrePrefix.plate.getMaterialAmount(Rubber) * 2), // plate (for the cable)
+                new MaterialStack(Polyethylene, OrePrefix.plate.getMaterialAmount(Polyethylene) * 2))); // plate
+
+        OreDictUnifier.registerOre(MetaTileEntities.HULL[GTValues.EV].getStackForm(), new ItemMaterialInfo(
+                new MaterialStack(Titanium, OrePrefix.plate.getMaterialAmount(Titanium) * 8), // plate
+                new MaterialStack(Aluminium, OrePrefix.cableGtSingle.getMaterialAmount(Aluminium) * 2), // single cable
+                new MaterialStack(Rubber, OrePrefix.plate.getMaterialAmount(Rubber) * 2), // plate (for the cable)
+                new MaterialStack(Polyethylene, OrePrefix.plate.getMaterialAmount(Polyethylene) * 2))); // plate
+
+        OreDictUnifier.registerOre(MetaTileEntities.HULL[GTValues.IV].getStackForm(), new ItemMaterialInfo(
+                new MaterialStack(TungstenSteel, OrePrefix.plate.getMaterialAmount(TungstenSteel) * 8), // plate
+                new MaterialStack(Platinum, OrePrefix.cableGtSingle.getMaterialAmount(Platinum) * 2), // single cable
+                new MaterialStack(Rubber, OrePrefix.plate.getMaterialAmount(Rubber) * 2), // plate (for the cable)
```

Comment:
IV and above cables cannot be made with rubber so these recipes are weird

Reply:
The rubber here is for the coating for the platinum cables

Reply:
platinum cable has no Rubber coating recipe
it can only be coated by Silicone Rubber or Styrene-Butadiene Rubber...